src/helpers/editOptionTypes.ts

```diff
@@ -0,0 +1,8 @@
+enum EditActions {
```

Comment:
I would expect this to cause issues in `CoinDivider.js` and `ChartContextButton.js` which are both JS files
Until we convert those to TS, we can follow the pattern in exporting both an enum as well as the normal JS object

example file that does this: `helpers/networkTypes.ts`

our pattern is to call the enum by singular name: e.g `EditAction` here and for the default export to still be used as `EditActions` for JS files. This will need an update to `useCoinListEditOptions.ts` to `import { EditAction }`

Reply:
Actually, we can move to this new pattern instead: https://rainbowhaus.slack.com/archives/C02L6B0H57S/p1639357314056100